src/platform/OpenThread/GenericThreadStackManagerImpl_OpenThread.cpp

```diff
@@ -1896,6 +1896,10 @@ CHIP_ERROR GenericThreadStackManagerImpl_OpenThread<ImplClass>::_RequestSEDActiv
 
     if (!onOff && delayIdle && CHIP_DEVICE_CONFIG_SED_ACTIVE_THRESHOLD.count() != 0)
     {
+        // StartTimer will cancel a timer if the same callback & context is used.
+        // This will have for effect to cancel the previous one (if any) and start
+        // a new timer of the same duration. Thus effectively prolonging the active threshold
+        // without consuming aditionnal resources.
```

Comment:
```suggestion
        // This will have the effect of canceling the previous one (if any) and starting
        // a new timer of the same duration. This effectively prolongs the active threshold
        // without consuming additional resources.
```